iOSTransport/ALXAppDelegate.m

```diff
@@ -0,0 +1,49 @@
+//
+//  AppDelegate.m
+//  iOSTransport
+//
+//  Created by Alex De Souza Campelo Lima on 3/5/15.
+//  Copyright (c) 2015 Alex De Souza Campelo Lima. All rights reserved.
+//
+
+#import "ALXAppDelegate.h"
+
+@interface ALXAppDelegate ()
+
+@end
+
+@implementation ALXAppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    // Override point for customization after application launch.
+    [[UINavigationBar appearance] setTintColor:[UIColor whiteColor]];
+    [[UITabBar appearance] setTintColor:[UIColor whiteColor]];
+    
+    [GMSServices provideAPIKey:@"AIzaSyBgHchl4o7ErtJEIkAz7UTlFR0lZorU5Io"];
```

Comment:
The values were replaced by constants. So this issue was solved on commit c47eed3c51025c6269663774849ed6c68cc14879